src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## Unreleased
 
+* Following attributes are available while sampling:
+  * `http.request.method`,
+  * `server.address`,
+  * `server.port`,
+  * `url.scheme`,
+  * `url.path`,
+  * `user_agent.original`,
+  * `url.query`.
```

Comment:
Swap these two for A-Z sorting?

Reply:
4252de94349a3db5b7c59c52dfe87c1d831870fc